assets/i18n/qqq.json

```diff
@@ -351,11 +351,22 @@
   "I18N_DONATE_PAGE_CONTENT_HEADING_1": "Heading of the first section of the donate page.",
   "I18N_DONATE_PAGE_CONTENT_HEADING_2": "Heading of the second section of the donate page.",
   "I18N_DONATE_PAGE_CONTENT_HEADING_3": "Heading of the third section of the donate page.",
+	"I18N_DONATE_PAGE_CONTENT_HEADING_4": "Heading of the fourth section of the donate page.",
   "I18N_DONATE_PAGE_CONTENT_HIGHTLIGHTS": "Heading of the highlights section.",
   "I18N_DONATE_PAGE_CONTENT_HIGHTLIGHTS_CONTENT_1": "Highlights content 1.",
   "I18N_DONATE_PAGE_CONTENT_HIGHTLIGHTS_CONTENT_2": "Highlights content 2.",
   "I18N_DONATE_PAGE_CONTENT_HIGHTLIGHTS_TITLE_1": "Highlights title 1",
   "I18N_DONATE_PAGE_CONTENT_HIGHTLIGHTS_TITLE_2": "Highlights title 2",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_1": "Birthplace of learner 1",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_2": "Birthplace of learner 2",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_3": "Birthplace of learner 3",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_4": "Birthplace of learner 4",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_5": "Birthplace of learner 5",
+	"I18N_DONATE_PAGE_CONTENT_LEARNER_QUOTE_1": "Quote from learner 1",
```

Comment:
Since this is an "instructions for translators" file, should we say something about "please include the quotation marks at the beginning and end of the quote in translations", do you think?

Reply:
@amyyeung17 Just a note, please don't resolve conversations, it makes it harder to do follow-up reviews! Instead, if you've done something, please reply to the comment with "Done". See instructions in step 5 [here](https://github.com/oppia/oppia/wiki/Make-a-pull-request). Thanks!